content/yaml-quick-start/relocating from AppCenter to Codemagic.md

```diff
@@ -0,0 +1,706 @@
+---
+title: Seamless migration from App Center to Codemagic
+description: How to ship your workflows to Codemagic
+weight: 1
+---
+
+## Quick comparison
+
+
+| **Features**                | **App Center** | **Codemagic**|
+|-----------------------------| ---------------|--------------|
+| `Swift/Objective-C iOS`     | `Yes`          | `Yes`        |
```

Comment:
as some visual improvement to better show the difference you can use emoji.
☑️✅ (gray for App Center, green for Codemagic)
and
❌✅